clang/utils/TableGen/ClangAttrEmitter.cpp

```diff
@@ -5178,6 +5178,9 @@ GetAttributeHeadingAndSpellings(const Record &Documentation,
 
 static void WriteDocumentation(const RecordKeeper &Records,
                                const DocumentationData &Doc, raw_ostream &OS) {
+  if (const StringRef Label = Doc.Documentation->getValueAsString("Label");
```

Comment:
```suggestion
  if (StringRef Label = Doc.Documentation->getValueAsString("Label");
```
I think you may have missed this